lms/services/vitalsource/service.py

```diff
@@ -1,41 +1,60 @@
 from typing import List
 
 from lms.services.vitalsource._client import VitalSourceClient
+from lms.services.vitalsource.exceptions import VitalSourceError
 from lms.services.vitalsource.model import VSBookLocation
 
 
 class VitalSourceService:
     """A high-level interface for dealing with VitalSource."""
 
-    def __init__(self, client: VitalSourceClient, enabled):
+    user_lti_param = None
+    """The LTI parameter to use to work out the LTI user."""
+
+    def __init__(self, client: VitalSourceClient, enabled, user_lti_param):
+        """
+        Instantiate the service.
+
+        :param client: VitalSource client for connecting to the API
+        :param enabled: Are we enabled at all?
+        :param user_lti_param: Which LTI parameter to read to get the user
+            reference
+        """
+        self._client = client
         self._enabled = enabled
-        self.client = client
+        self.user_lti_param = user_lti_param
 
     @property
     def enabled(self) -> bool:
         """Check if the service has everything it needs to work."""
 
-        return bool(self._enabled and self.client)
+        return bool(self._enabled and self._client and self.user_lti_param)
```

Comment:
Is `user_lti_param` required for any type of VS integration?

Reply:
It was, but isn't any more.